docker-compose.yml

```diff
@@ -112,42 +112,42 @@ services:
     ports:
       - "8000:8000"
 
-  cirrus:
-    build:
-      context: cirrus/server/
-      dockerfile: Dockerfile
-    env_file: .env
-    links:
-      - kinto
-    volumes:
-      - ./cirrus/server/:/cirrus
-    working_dir: /cirrus
-    ports:
-      - "8001:8001"
-  demo-app-server:
-    build:
-      context: ./demo-app/server
-      dockerfile: Dockerfile
-    ports:
-      - '3002:3002'
-    restart: always
-    links:
-      - cirrus
-    environment:
-      - CIRRUS=http://cirrus:8001
+  # cirrus:
+  #   build:
+  #     context: cirrus/server/
+  #     dockerfile: Dockerfile
+  #   env_file: .env
+  #   links:
+  #     - kinto
+  #   volumes:
```

Comment:
I see cirrus stuff commented out, is there a reason?

Reply:
accidental commit. it prevents building cirrus, which without hydrobuild was taking a long time